packages/api/src/apis/collectionsApi/endpoints/getIndividualGroups/e2e.spec.js

```diff
@@ -5,7 +5,7 @@ const { fullFormExample } = require('../../testData/individualGroup')
 
 const validCatalogNumber = '123456'
 const validTaxonName = 'Chironectes minimus'
-const invalidCatalogNumber = '234567'
+const invalidCatalogNumber = 'Some catalogNumber'
 const invalidTaxonName = 'Some taxa'
```

Comment:
might be more clear to use "invalid" instead of "some" (because it's not clear with "some" that it's invalid)